Makefile

```
@@ -0,0 +1,24 @@
+.PHONY: typecheck lint format test check clean help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)